var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,290 @@
+            env['TF_CUDA_COMPUTE_CAPABILITIES'] = "6.1,7.5"
+            env['NCCL_INSTALL_PATH'] = str(spec['nccl'].prefix.lib)
+            env['NCCL_HDR_PATH'] = str(spec['nccl'].prefix.include)
+            env['TF_CUDA_CLANG'] = '0'
+            env['TF_NEED_ROCM'] = '0'
+            env['TF_NEED_TENSORRT'] = '0'
+        else:
+            env['TF_NEED_CUDA'] = '0'
+            env['TF_CUDA_VERSION'] = ''
+            env['CUDA_TOOLKIT_PATH'] = ''
+            env['TF_CUDNN_VERSION'] = ''
+            env['CUDNN_INSTALL_PATH'] = ''
+
+        # config options for version 1.0
+        if self.spec.satisfies('@1.0.0-rc2:'):
+            env['CC_OPT_FLAGS'] = '-march=x86-64 -mtune=generic'
+            env['TF_NEED_JEMALLOC'] = '0'
+            env['TF_NEED_HDFS'] = '0'
+            env['TF_ENABLE_XLA'] = '0'
+            env['PYTHON_LIB_PATH'] = self.module.site_packages_dir
+            env['TF_NEED_OPENCL'] = '0'
+
+        # additional config options starting with version 1.2
+        if self.spec.satisfies('@1.2.0:'):
+            env['TF_NEED_MKL'] = '0'
+            env['TF_NEED_VERBS'] = '0'
+
+        # additional config options starting with version 1.3
+        if self.spec.satisfies('@1.3.0:'):
+            env['TF_NEED_MPI'] = '0'
+
+        # additional config options starting with version 1.5
+        if self.spec.satisfies('@1.5.0:'):
+            env['TF_NEED_S3'] = '0'
+            env['TF_NEED_GDR'] = '0'
+            env['TF_NEED_OPENCL_SYCL'] = '0'
+            env['TF_SET_ANDROID_WORKSPACE'] = '0'
+            # env variable is somehow ignored -> brute force
+            filter_file(r'if workspace_has_any_android_rule\(\)',
+                        r'if True',
+                        'configure.py')
+
+        # additional config options starting with version 1.6
+        if self.spec.satisfies('@1.6.0:'):
+            env['TF_NEED_KAFKA'] = '0'
+
+        # additional config options starting with version 1.8
+        if self.spec.satisfies('@1.8.0:'):
+            env['TF_DOWNLOAD_CLANG'] = '0'
+            env['TF_NEED_AWS'] = '0'
+
+        # additional config options starting with version 1.12
+        if self.spec.satisfies('@1.12.0:'):
+            env['TF_NEED_IGNITE'] = '0'
+            env['TF_NEED_ROCM'] = '0'
+
+        # set tmpdir to a non-NFS filesystem (because bazel uses ~/.cache/bazel)        # noqa: E501
+        # TODO: This should be checked for non-nfsy filesystem, but the current
+        #       best idea for it is to check
+        #           subprocess.call(['stat', '--file-system', '--format=%T', tmp_path]) # noqa: E501
+        #       to not be nfs. This is only valid for Linux and we'd like to
+        #       stay at least also OSX compatible
+        tmp_path = env.get('SPACK_TMPDIR', '/tmp/spack') + '/tf'
+        mkdirp(tmp_path)
+        env['TEST_TMPDIR'] = tmp_path
+        env['HOME'] = tmp_path
+        env['CC'] = env['SPACK_CC']
```

Comment:
check if this is taken care of by the bazel changes

Reply:
now package compiles just fine without those. So, I removed them.